build/templates/_grpc_stub_interpreter.py.mako

```diff
@@ -33,7 +35,13 @@ class GrpcStubInterpreter(object):
         self._grpc_options = grpc_options
         self._lock = threading.RLock()
         self._client = ${module_name}_grpc.${service_class_prefix}Stub(grpc_options.grpc_channel)
-        self._${config['session_handle_parameter_name']} = 0
+        self.set_session_handle()
+
+    def set_session_handle(self, value=session_grpc_types.Session()):
+        self._${config['session_handle_parameter_name']} = value
```

Comment:
Does this still default initialize to 0? I can't tell. Does it need to?

Reply:
Yes, the id in the default-constructed session_grpc_types.Session() object is 0. We test this on line 88 of [this test](https://github.com/ni/nimi-python/blob/9201bc8259777992b502427c926c15bb901115e1/src/nifake/unit_tests/test_grpc.py#L84).